### ISSUE TITLE
Request: WalletConnect

### ISSUE THREAD

**Issue by 0xr2m** [interep-project/reputation-service#111]
Hi - would like to state that adding WalletConnect could be useful, as it allows for multisigs to join.  This widens the set of high-reputation accounts that could use Interep, as they may currently be co-owned by teams and small groups.  Thanks.

**Comment by glamperd** [comment]
Yes, I think this might be a useful addition. Note that using a multisig account could be tricky in some cases (e.g. linking it to an individual's social ID). We'll leave that to app designers to be aware of.
